src/pages/departures/[[...id]].tsx

```diff
@@ -61,6 +62,16 @@ export const getServerSideProps = withGlobalData(
     DeparturesLayoutProps & DeparturesContentProps,
     { id: string[] | undefined }
   >(async function ({ client, params, query }) {
+    if (query?.id && !params?.id) {
+      const id = query.id;
+      delete query.id;
```

Comment:
Does it matter if it's deleted or not?

Reply:
I does not matter really, but I though it would be cleaner to remove it from the query